src/library/scala/io/Source.scala

```diff
@@ -167,6 +167,10 @@ object Source {
 
   def fromInputStream(is: InputStream)(implicit codec: Codec): BufferedSource =
     createBufferedSource(is, reset = () => fromInputStream(is)(codec), close = () => is.close())(codec)
+
+  def fromClassPath(resource: String, classLoader: ClassLoader = ClassLoader.getSystemClassLoader)(implicit codec: Codec): BufferedSource =
```

Comment:
I know it's "just" a default, but I'm wondering if the system class loader is the right default. When loading resources most frameworks use the context classloader (which defaults to the system one anyway).